backend/docs/manual/3-administrator/3.1-email-templates.rst

```diff
@@ -20,6 +20,27 @@ Hier kun je de onderwerpen en de emailteksten voor de emails die worden gestuurd
 - De record manager als een reviewer aanpassingen heeft aangevraagd op een lijst.
 - De record manager en de reviewers als een lijst succesvol is vernietigd.
 
+Tekst en HTML tablonen moeten geconfigureerd worden voor elke email. Een voorbeeld voor de email
+die wordt gestuurd naar een reviewer als een lijst goedgekeurd moet worden zou kunnen is:
```

Comment:
```suggestion
die wordt gestuurd naar een beoordelaar als een lijst goedgekeurd moet worden zou kunnen is:
```

I think we should use consistent terminology, so we might need to figure out what whe use where?